ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -227,6 +288,176 @@ def context_check_botmessage(botmessage):
             True
         """
 
-        if "name" in botmessage:
+        if "name" in botmessage or u'नाम' in botmessage:
             return True
         return False
+
+    def get_hindi_names_from_regex(self, text):
+        """
+        This method is used to detect hindi names which obey the regexes
+        Args:
+            text (str): text from which hindi names obeying the regex have to be extracted
+
+        Returns:
+            detect_text_lists (tuple): two dimensional tuple
+            1. text_list (list): representing the detected text
+            2. text_list (list): representing the original text
+
+        Examples:
+            text = u'मेरा नाम प्रतिक श्रीदत्त जयराओ है'
+            get_hindi_text_from_regex(text=text)
+            >>([u'प्रतिक', u'श्रीदत्त', u'जयराओ'], [u'प्रतिक', u'श्रीदत्त', u'जयराओ'])
+
+        """
+        text_list = self.get_hindi_text_from_regex(text=text)
+
+        detected_names = []
+        if text_list:
+            text_list = [self.replace_stopwords_hindi(text=x) for x in text_list]
+            for each in text_list:
+                detected_names.extend(each.split())
+        text_list = detected_names
+
+        return text_list, text_list
+
+    def get_hindi_names_without_regex(self, text):
```

Comment:
why can't we  do things in same function instead of making `get_devnagri_text(text=text)`
`self.replace_detected_text((original_text_list, original_text_list))`
`self.detect_person_name_entity(replaced_text=replaced_text)`

Reply:
Done